### ISSUE TITLE
Clicking evCert label should toggle siteInfoDialog

### ISSUE THREAD

**Issue by jonathansampson** [brave/browser-laptop#14611]
### Description
Clicking the `span.evCert` element should toggle visibility of the `siteInfoDialog` element (as it does when you click the lock icon). This is currently how Chrome, Firefox, and Edge work.

### Steps to Reproduce
1. Navigate to github.com in Brave
2. Click "GitHub, Inc. [US]" in the addressbar

**Actual result:**
Nothing happens.

**Expected result:**
`siteInfoDialog` element is shown.

**Reproduces how often:**
Always.

### Brave Version (about:brave info):
``` 
Brave: 0.23.19 
V8: 6.7.288.46 
rev: 178c3fbc045a0cbdbe098db08307503cce952081 
Muon: 7.1.3 
OS Release: 10.0.17134 
Update Channel: Release 
OS Architecture: x64 
OS Platform: Microsoft Windows 
Node.js: 7.9.0 
Tor: 0.3.3.7 (git-035a35178c92da94) 
Brave Sync: v1.4.2 
libchromiumcontent: 67.0.3396.87
```

**Reproducible on current live release:**
Yes.

### Additional Information
There should probably be a subtle hover-effect when mousing over the `span.evCert` element and lock icon.

**Comment by srirambv** [comment]
This is same as https://github.com/brave/browser-laptop/issues/7888?

**Comment by bsclifton** [comment]
@srirambv this is indeed a dupe of https://github.com/brave/browser-laptop/issues/7888.  This is fixed with 0.24.x 😄👍 via https://github.com/brave/browser-laptop/pull/13164
